packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -72,7 +72,8 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implement
         }),
         chain: this.getType(),
         chainSpecific: {
-          sequence: data.sequence
+          accountNumber: data.accountNumber.toString(),
```

Comment:
have you tested this with a cosmos account that has never transacted? watch out for undefined accountNumber for new accounts. possible it's an empty string in api so no harm

Reply:
No I haven't tested that use case. I'll test that.